packages/core/src/sync/events.ts

```diff
@@ -500,7 +500,7 @@ export const decodeEvents = (
 
                 event: {
                   name: safeName,
-                  args,
+                  args: removeNullCharacters(args),
```

Comment:
@kyscott18 lmk if this is the correct place to do this - both historical and realtime events pass through this, yes?

Reply:
Yes